sparse_strips/vello_api/src/flatten.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright 2025 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Types for flattening curves.
+
+/// A point.
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub struct Point {
+    /// The x coordinate of the point.
+    pub x: f32,
+    /// The y coordinate of the point.
+    pub y: f32,
+}
```

Comment:
If my reasoning about `Tile` and `Strip` living in `sparse_strips/vello_common` is sound, `Point` and `FlatLine` should probably also live there.